packages/cms/lib/modules/participatory-budgeting-widgets/index.js

```diff
@@ -86,16 +86,33 @@ module.exports = {
                 }
             }) : [];
 
-            widget.formatImageUrl = function (url, width, height, crop, location) {
-                if (url) {
-                    url = url + '/:/rs=w:' + width + ',h:' + height;
-                    url = crop ? url + ';cp=w:' + width + ',h:' + height : url;
-                } else {
-                    url = '/modules/openstad-assets/img/placeholders/idea.jpg';
+            widget.getImageForIdea = function(idea, widget, width, height, crop) {
+                const defaultImageUrl = '/modules/openstad-assets/img/placeholders/idea.jpg';
+
+                const boundFormatImageUrl = (image) => formatImageUrl(image, width, height, crop);
+
+                if (idea.extraData && idea.extraData.images && idea.extraData.images.length) {
+                    return boundFormatImageUrl(idea.extraData.images[0]);
+                }
+
+                if (idea.extraData && idea.extraData.theme && widget.themes.length > 0) {
+                    const themeObject = widget.themes.find(theme => theme.value === idea.extraData.theme);
```

Comment:
Waar wordt `widget.themes` gevuld? Zit de `themes` die je wilt niet in de `global`?